tests/models/test_fastercnn_creation.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from mantisshrimp.models.mantis_rcnn import *
+import torch
+
+
+def test_fastercnn():
+    # We need to instantiate with all possible combinations
+    # Taken directly from mantis_faster_rcnn
+    supported_resnet_models = [
```

Comment:
Yeah, again, let's figure out a way of defining those in the model

Reply:
Defined as a class method with self inside mantis_faster_rcnn